tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 576x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ranks, dap_samples = run_sbc(thetas, xs, posterior_, num_workers=1)\n",
+    "check_stats = check_sbc(ranks, thetas, dap_samples, 1_000)\n",
+    "print(check_stats)\n",
+    "f, ax = sbc_rank_plot(ranks, 1_000, plot_type = \"hist\", num_bins=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "The histogram of ranks now shoots above the allowed (greyed) area for a uniform distributed around the extrema. We made the posterior samples too thin, so we received more extreme counts of ranks. The KS p-values vanish again and the `c2st` metric of the ranks is also larger than `.5` which underlines that our rank distribution is not uniformly distributed."
```

Comment:
Again, very nice to show both cases. And again it would be great to conclude with a key take away how SBC rank histograms can be used to diagnose the concrete bias in the estimator.

Reply:
34c9cf2